engineering/2022_02_24_sharable_tensorboards/README.md

```diff
@@ -0,0 +1,33 @@
+# Vertex AI TensorBoard alternative for smaller budgets
+
+At the time of writing the pricing plan for Vertex AI TensorBoard is the following: 
+> Vertex AI TensorBoard charges a monthly fee of $300 per unique active user. Active users are measured through the Vertex AI TensorBoard UI.
+
+This can get pretty expensive if you want to share your experiment tracking results with your team...
+
+## Any other alternatives? 🤷‍♂️
+
+Some other options (like TensorBoard.dev)  make your data publicly available (it goes without saying but this is a big nono when working with customer data 🙅‍♂️)
+Other solutions require other people to set up a local environment to run TensorBoard and thus require certain technical skills from those people.
+
+## Our solution 🦾
+
+You can use the Terraform code in the `tensorboard.tf` file to generate your own securely sharable TensorBoard for a fraction of the price. 
+
+Some minor changes have to be made to the file: 
```

Comment:
These are now stored in variables, so the description here isn't entirely accurate. I think we can just reword to something like "There are some variables you have to supply:" - WDYT?

Reply:
Nice catch! Added the change just now!